src/strings/en-us.json

```diff
@@ -147,6 +147,7 @@
     "ChannelNumber": "Channel number",
     "Channels": "Channels",
     "CinemaModeConfigurationHelp": "Cinema mode brings the theater experience straight to your living room with the ability to play trailers and custom intros before the main feature.",
+    "ClientSettings": "Client Settings",
```

Comment:
Maybe “App Settings” would be more clear?

Reply:
I tried to keep it generic since some platforms don't use the term app, such as desktops.